numpy/core/numeric.py

```diff
@@ -2089,15 +2089,12 @@ def indices(dimensions, dtype=int):
     """
     dimensions = tuple(dimensions)
     N = len(dimensions)
-    if N == 0:
-        return array([], dtype=dtype)
+    shape = (1,)*N
     res = empty((N,)+dimensions, dtype=dtype)
     for i, dim in enumerate(dimensions):
```

Comment:
Yes we do worry about line length, we are kind of picky about it, although there are some exceptions. Also about spaces around `+ - =', etc., but not around `**` and somewhat optional around `*/`. Other style nits we sometimes ignore in tests for readabilty. Use your judgement about style, but lean towards picky as style deviations otherwise accumulate, and also because we may want to run a style checker for violations in the future. SciPy already does that.